pkg/analyzer/deployment.go

```diff
@@ -45,14 +47,25 @@ func (d DeploymentAnalyzer) Analyze(a common.Analyzer) ([]common.Result, error)
 	AnalyzerErrorsMetric.DeletePartialMatch(map[string]string{
 		"analyzer_name": kind,
 	})
-
-	deployments, err := a.Client.GetClient().AppsV1().Deployments(a.Namespace).List(context.Background(), v1.ListOptions{})
-	if err != nil {
-		return nil, err
+	var list *appsv1.DeploymentList = &appsv1.DeploymentList{}
+	if len(a.Resources["Deployment"]) > 0 {
```

Comment:
Can we return error on all other analyzers that does not yet support "resources"? 🤔 otherwise it might be bit misleading.